### PR TITLE
feat(create-router): add redirectPage method to replace current state in history

### LINKED ISSUE #19: createRouter support history replaceState
Sometimes we need to replace latest state in history (aka redirect).
I am proposing following API changes:
```ts
router.open(path: string, redirect?: boolean);

// additional export that will pass flag to open
export function redirectPage(router, name, params);
```
It increases bundle-size on 13 B.

Please approve or give another suggestions and I will prepare PR.

### PR DESCRIPTION
**Pull request by Guria**
fixes #19

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -481,3 +481,12 @@ onExit() {
   openPage(router, 'home')
 }
 ```
+
+If you need to replace current URL programmatically you can use `replacePage`:
+
+```ts
+onExit() {
```
Comment (ai) [inline README.md:488]: It is not cool that we use the same example in a different use cases. It is better to show a good use case for `replacePage` or change `openPage` example (if `replacePage` is the best for exit).

File: clean-stores/index.d.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { MapBuilder, AnySyncBuilder } from '../define-map/index.js'
-import { Store } from '../create-store/index.js'
+import { StoreLike } from '../create-store/index.js'
```
Comment (ai) [inline clean-stores/index.d.ts:2]: You need to rebase changes since I moved to `ReadableStore`.

File: package.json [hunk 3]
```diff
@@ -12,7 +12,7 @@
     "store"
   ],
   "scripts": {
-    "test": "node --experimental-vm-modules node_modules/.bin/jest --coverage && eslint . && check-dts && size-limit && yaspeller *.md"
+    "test": "node --experimental-vm-modules node_modules/.bin/jest && eslint . && check-dts && size-limit && yaspeller *.md"
```
Comment (ai) [inline package.json:15]: We can’t remove coverage. I use it to be sure that I tested everything.

What part of the code you can’t cover?

Comment (Guria) [reply package.json:15]: sorry, leaked from local testing

**Comment by ai** [comment]
Fix docs and we are ready to merge.

File: README.md (hunk 1 above)
Comment (Guria) [reply README.md]: fixed

File: README.md [hunk 4]
```diff
@@ -481,3 +481,16 @@ onExit() {
...
+
+If you need to replace current URL programmatically you can use `replacePage`:
+
+```ts
+login() {
+  openPage(router, 'login')
+}
+
+onLoginSuccess() {
+  # replace login route, so we don't face it on back navigation
```
Comment (ai) [inline README.md:493]:
```suggestion
  // Replace login route, so we don’t face it on back navigation
```

**Comment by Guria** [comment]
@ai fixed

**Comment by ai** [comment]
I will release these changes today or tomorrow together with renaming project to `nanostores`

**Pull request merged**